src/test/java/org/broadinstitute/consent/http/resources/DataAccessRequestResourceTest.java

```diff
@@ -418,11 +419,27 @@ void testPostProgressReportUserNotAuthorized() throws IOException {
     Pair<InputStream, FormDataContentDisposition> collabFile = mockFormDataMultiPart("collab.txt");
     Pair<InputStream, FormDataContentDisposition> ethicsFile = mockFormDataMultiPart("ethics.txt");
     initResource();
+    user.setLibraryCards(null);
 
-    assertThrows(ForbiddenException.class, () -> {
-      resource.postProgressReport(authUser, "", "", collabFile.getLeft(),
-          collabFile.getRight(), ethicsFile.getLeft(), ethicsFile.getRight());
-    });
+    Response response = resource.postProgressReport(authUser, "", "", collabFile.getLeft(),
+        collabFile.getRight(), ethicsFile.getLeft(), ethicsFile.getRight());
+    assertEquals(HttpStatusCodes.STATUS_CODE_FORBIDDEN, response.getStatus());
+  }
+
+  @Test
+  void testPostProgressReportCollabAndEthicsFilesWithoutLibraryCardsThrows() throws IOException {
+    when(userService.findUserByEmail(user.getEmail())).thenReturn(user);
+    DataAccessRequest parentDar = generateDataAccessRequest();
+    parentDar.getData().setCollaborationLetterLocation("collaborationLetterLocation");
+    parentDar.getData().setIrbDocumentLocation("irbDocumentLocation");
+    when(dataAccessRequestService.findByReferenceId(any())).thenReturn(parentDar);
+    Pair<InputStream, FormDataContentDisposition> collabFile = mockFormDataMultiPart("collab.txt");
+    Pair<InputStream, FormDataContentDisposition> ethicsFile = mockFormDataMultiPart("ethics.txt");
```

Comment:
This would be a nice place to use `var` IMO
```suggestion
    var collabFile = mockFormDataMultiPart("collab.txt");
    var ethicsFile = mockFormDataMultiPart("ethics.txt");
```